packages/framework/src/client.ts

```diff
@@ -679,8 +679,16 @@ export class Client {
     try {
       const templateString = this.preprocessTranslationPatterns(JSON.stringify(templateControls));
       const parsedTemplate = this.templateEngine.parse(templateString);
+      const discoveredWorkflow = this.getWorkflow(event.workflowId);
 
       const renderVariables = {
+        workflow: {
+          workflowId: discoveredWorkflow.workflowId,
+          name: discoveredWorkflow.name,
+          description: discoveredWorkflow.description,
+          tags: discoveredWorkflow.tags,
+          severity: discoveredWorkflow.severity,
+        },
```

Comment:
added the new `workflow` variable to the variables that are passed to template engine for the compilation/rendering